Cargo.toml

```diff
@@ -16,7 +16,6 @@ keywords = ["docker"]
 # OpenSSL is fairly hard to build on certain platforms, especially if you
 # want to produce release binaries.  So we enable it by default, but allow
 # it to be excluded using `default-features = false`.
-default = ["ssl"]
```

Comment:
This disables SSL by default, and it a breaking API change for other users of the library. What was your rationale for this?